src/nls/cs/strings.js

```diff
@@ -410,9 +422,11 @@ define({
     "CMD_TWITTER"                         : "{TWITTER_NAME} - Twitter",
     "CMD_ABOUT"                           : "O aplikaci {APP_TITLE}",
     "CMD_OPEN_PREFERENCES"                : "Otevřít soubor s preferencemi",
+    "CMD_OPEN_KEYMAP"                     : "Otevřít uživatelské klávesové zkratky",
 
     // Řetězce pro main-view.html
     "EXPERIMENTAL_BUILD"                   : "experimentální verze",
+    "RELEASE_BUILD"                        : "verze",
```

Comment:
@dvorapa Just to explain the process... Ideally we'd like another native speaker to review translation work.  If no one is able to review, then we use Google Translate to check for typos and other obvious errors the best we can.

In this case since most of these translations came from @kvarel, I think _you_ basically count as the reviewer anyway (assuming you looked over all these new strings) -- so we maybe could have skipped that step.